<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiredConnectionWidget</class>
 <widget class="QWidget" name="WiredConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Device MAC address:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="macAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string notr="true">HH:HH:HH:HH:HH:HH;_</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Cloned MAC address:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="clonedMacAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string notr="true">HH:HH:HH:HH:HH:HH;_</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>MTU:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KIntSpinBox" name="mtu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>Automatic</string>
     </property>
     <property name="suffix">
      <string> bytes</string>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="speedLabel">
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KIntSpinBox" name="speed">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>Automatic</string>
     </property>
     <property name="suffix">
      <string> Mbit/s</string>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="duplexLabel">
     <property name="text">
      <string>Duplex</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="duplex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Full</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Half</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="autonegotiate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Autonegotiate</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>118</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>macAddress</tabstop>
  <tabstop>clonedMacAddress</tabstop>
  <tabstop>mtu</tabstop>
  <tabstop>speed</tabstop>
  <tabstop>duplex</tabstop>
  <tabstop>autonegotiate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>autonegotiate</sender>
   <signal>toggled(bool)</signal>
   <receiver>speedLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autonegotiate</sender>
   <signal>toggled(bool)</signal>
   <receiver>speed</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autonegotiate</sender>
   <signal>toggled(bool)</signal>
   <receiver>duplexLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autonegotiate</sender>
   <signal>toggled(bool)</signal>
   <receiver>duplex</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
